packages/api/lib/writeRecords/write-granules.js

```diff
@@ -483,13 +494,25 @@ const _writeGranuleRecords = async (params) => {
         apiGranuleRecord.createdAt = pgGranule.created_at.getTime();
       }
 
+      if (writeConstraints && (pgGranule.status === 'running' || pgGranule.status === 'queued')) {
```

Comment:
Do we have a helper for determining if a granule's status is running or queued? 🤔

Reply:
I created the helper method, but this check is only happening the one time in this file, so I'm not sure of the utility unless it gets used more down the line. The check is also happening in the granule upsert in `packages/db/src/models/granule.ts`, but I'm not sure if we would want to import the utils.js file there.

Reply:
What are all the possible states? Could you do `!isStatusFinalState(status)`?

Reply:
I don't actually know all the possible states, or if they are listed anywhere. The ones I've seen and know of are 'completed', 'failed', 'running', and 'queued'. Searching through the repo I have not yet come across anything else, but unless I see some documentation saying these are the only options I'm not super confident. 
`isStatusFinalState()` checks against 'completed' and 'failed', so if there are only the four, I suppose it would work. However, I personally kind of prefer the explicitness of checking the statuses you want, doing `!isStatusFinalState()` would probably be fine, but you'd have to go and double check what that helper function is checking for if you're not that familiar. Don't feel _that_ strongly though, so if you want to insist I'd just want to make sure there are no other statuses first.